commands/reminder/setReminder.js

```diff
@@ -3,101 +3,101 @@ const { DateTime } = require('luxon');
 
 // TODO: needs permission 'MANAGE_CHANNELS'
 module.exports = {
-    data: new SlashCommandBuilder()
-        .setName('setreminder')
-        .setDescription('Sets a reminder')
-        .addStringOption(option =>
-            option.setName('deadline')
-                .setDescription('The deadline in CST in YYYY-MM-DD HH:MM format')
-                .setRequired(true))
-        .addChannelOption(option =>
-            option.setName('channel')
-                .setDescription('The channel to send the reminder')
-                .setRequired(true))
-        .addStringOption(option =>
-            option.setName('timeInAdvance')
-                .setDescription('The time before the deadline you would like the reminder to set off')
-                .addChoice('30 min', '30m')
-                .addChoice('1 hour', '1h')
-                .addChoice('1 day', '1d')
-                .setRequired(true))
-        .addStringOption(option =>
-            option.setName('reminderMessage')
-                .setDescription('The reminder message to send to channel')
-                .setRequired(true)),
-    async execute(interaction) {
-        const options = interaction.options;
-        const deadline = options.getString('deadline');
-        const channel = options.getChannel('channel');
-        const timeInAdvance = options.getString('timeInAdvance');
-        const message = options.getString('reminderMessage');
-        const cst = 'America/Chicago';
+  data: new SlashCommandBuilder()
+    .setName('setreminder')
+    .setDescription('Sets a reminder')
+    .addStringOption(option =>
+      option.setName('deadline')
+        .setDescription('The deadline in CST in YYYY-MM-DD HH:MM format')
+        .setRequired(true))
+    .addChannelOption(option =>
+      option.setName('channel')
+        .setDescription('The channel to send the reminder')
+        .setRequired(true))
+    .addStringOption(option =>
+      option.setName('time_in_advance')
```

Comment:
Command names can't be in camel case, they will throw an error when run. The rest of the changes in this file are related to changing the command names and spacing.